package.json

```diff
@@ -35,13 +35,15 @@
     "react-native-tab-view": "^2.13.0",
     "react-native-vector-icons": "^6.6.0",
     "rn-fetch-blob": "^0.12.0",
-    "sendbird": "^3.0.119"
+    "sendbird": "^3.0.119",
+    "sinon": "^9.0.2"
```

Comment:
Why is this dependency necessary? Sometimes errors say you need to download dependencies but that is rarely the case.

Reply:
I was trying to figure out how to do unit testing with prop-types, and sinon was a solution. We are no longer using sinon, so I will delete it. Great catch.